src/main/webapp/app/services/visualizations/iframe.service.js

```diff
@@ -10,7 +10,7 @@
     function GenerateIframe(VisualizationUtils, $rootScope, D3Utils, filterParametersService) {
         return {
             build: function (record, element, panel, isNotification) {
-                var iframeLink = VisualizationUtils.getPropertyValue(record.properties, 'I frame link');
+                var iframeLink = VisualizationUtils.getPropertyValue(record.properties, 'iFrame link');
```

Comment:
Can you put it in constant file iFrame link

Reply:
its come from database